driver/main.F90

```diff
@@ -36,6 +102,18 @@ program rdycore_f90
     PetscCallA(RDyInit(ierr))
 
     if (trim(config_file) /= trim('-help')) then
+
+      PetscCallA(PetscOptionsGetString(PETSC_NULL_OPTIONS, PETSC_NULL_CHARACTER, '-rain', rainfile, rain_specified, ierr))
+      PetscCallA(PetscOptionsGetString(PETSC_NULL_OPTIONS, PETSC_NULL_CHARACTER, '-bc', bcfile, bc_specified, ierr))
+
+      if (rain_specified) then
+        call opendata(rainfile, rain_vec, nrain)
```

Comment:
Should we be calling `VecGetArrayF90` on `rain_ptr` here? And on `bc_ptr` below?